codice/ddf
# DDF-4136 Remo Remove platform-solr-server-standalone

#### What does this PR do?
Removed platform-solr-server-standalone module and artifacts and moved schema to solr-schema. Removed exported test jar for catalog-core-solr

#### Who is reviewing it? 

@figliold
@ethantmanns 
@tyler30clemens 

#### Select relevant component teams: 
https://github.com/orgs/codice/teams/solr

#### Ask 2 committers to review/merge the PR and tag them here.

@clockard
@pklinef

#### How should this be tested?
- Verify itests pass on PR build
- Build distribution
- Test ingest and queries

#### Any background context you want to provide?
#### What are the relevant tickets?
[DDF-4136](https://codice.atlassian.net/browse/DDF-4136)

#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
